ui/src/components/questions/post.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Markdown from '../helpers/markdown';
+import moment from 'moment';
+import Certificate from '../utils/certificate';
+import * as User from '../../models/user';
+import Button from '../forms/button';
+import { UserContext } from '../../usercontext';
+import Trophy from '../utils/trophy';
+import { t } from 'ttag';
+import Dialog from '../utils/dialog';
+import InputField from '../forms/inputfield';
+
+class Post extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            votes: 0,
+            isDeleteDialogOpen: false,
+            isEditDialogOpen: false,
+            currentBody: '',
+            postingToServer: false
```

Comment:
maybe a better name?

Reply:
what's a better name when interacting with the server?